src/components/DelimitedList/DelimitedList.stories.js

```diff
@@ -29,7 +29,11 @@ storiesOf(components('DelimitedList'), module)
   .addDecorator(centered)
   .add(
     'Default',
-    () => <DelimitedList className="bx--type-body-long-01" {...storyProps()} />,
+    () => (
+      <div style={{ width: '360px' }}>
+        <DelimitedList {...storyProps()} />
```

Comment:
- We can add this `style` property directly to the component itself to avoid an unnecessary `div`
- Also, are there `width` values we can leverage from the Carbon layout package here? We need to standardize on a width value across Storybook

Reply:
Nice! That's an elegant solution and reinforces grid usage